components/ContributionConfirmationModal.js

```diff
@@ -0,0 +1,237 @@
+
+  useEffect(() => {
+    setNetAmount(amountReceived - platformTip);
+  }, [amountReceived, platformTip]);
+
+  const triggerAction = async () => {
+    // Prevent submitting the action if another one is being submitted at the same time
+    if (loading) {
+      return;
+    }
+
+    const orderConfirmationDetails = {
+      totalAmount: { valueInCents: amountReceived, currency },
+      paymentProcessorFeesAmount: { valueInCents: paymentProcessorFee, currency },
+      platformTipAmount: { valueInCents: platformTip, currency },
+    };
+    try {
+      await confirmOrder({ variables: { id: order.id, action: 'MARK_AS_PAID', details: orderConfirmationDetails } });
+      addToast({
+        type: TOAST_TYPE.SUCCESS,
+        message: intl.formatMessage({ defaultMessage: 'Order confirmed successfully' }),
+      });
+      onClose();
+    } catch (e) {
+      addToast({ type: TOAST_TYPE.ERROR, message: i18nGraphqlException(intl, e) });
+    }
+  };
+
+  const handleChange = (fieldName, value) => {
+    if (fieldName === 'amountReceived') {
+      setAmountReceived(value);
+    } else if (fieldName === 'platformTip') {
+      setPlatformTip(value);
+    } else if (fieldName === 'paymentProcessorFee') {
+      setPaymentProcessorFee(value);
+    }
+  };
+
+  return (
+    <Modal width="578px" show={show} onClose={onClose} trapFocus>
+      <ModalHeader>
+        <FormattedMessage
+          defaultMessage="Confirm contribution to {collectiveName}"
+          values={{
+            collectiveName: order.toAccount.name,
+          }}
+        />
+      </ModalHeader>
+      <ModalBody>
+        <P mb={4} fontSize="13px">
+          <FormattedMessage defaultMessage="Confirm the amount of funds you have received in your host account." />
+        </P>
+        <Container mt="58px">
+          <Flex justifyContent="space-between" alignItems={['left', 'center']} flexDirection={['column', 'row']}>
+            <Span fontSize="14px" lineHeight="20px" fontWeight="700">
+              <FormattedMessage
+                defaultMessage="Amount initiated by {contributor}"
+                values={{ contributor: order.fromAccount.name }}
+              />
+            </Span>
+            <Box width="142px">
+              <InputField
+                className="horizontal"
+                name="amountInitiated"
+                type="currency"
+                onChange={() => null}
+                pre={getCurrencySymbol(currency)}
+                value={amountInitiated}
+                disabled
+              />
+            </Box>
+          </Flex>
+        </Container>
+        <StyledHr borderStyle="solid" mt="16px" mb="16px" />
+        <Container>
+          <Flex justifyContent="space-between" alignItems={['left', 'center']} flexDirection={['column', 'row']}>
+            <Span fontSize="14px" lineHeight="20px" fontWeight="400">
+              <FormattedMessage
+                defaultMessage="Amount received by {collective}"
```

Comment:
TBC with design, but I think what we want to display here is "Amount received by {fiscalHost}"

Reply:
Here, the amount received could be for any collective not necessarily for a fiscal host. Note that we use, `order.toAccount`; 

https://github.com/opencollective/opencollective-frontend/blob/96aa9e7936694003e12f95d15ed5853742b73b9c/components/ContributionConfirmationModal.js#L110-L113

For example this is from a user to a collective (not a fiscal host). 🤔 

![image](https://user-images.githubusercontent.com/12435965/146407671-1009b705-385f-40f1-80de-a3f2b001df4d.png)

Reply:
I guess what buggers me is that the amount is not received by the collective, but by the fiscal host (because they're the ones dealing with the bank account). Not a blocker though.

Reply:
@Betree : Oh I see what you mean. Let me post this on the issue so that design can weigh in on it and then if that's the case I'll reiterate on this in a separate issue. https://github.com/opencollective/opencollective/issues/4069#issuecomment-997947162

Reply:
@Betree is right, but maybe we can rephrase `"Amount received for {collective}"`.

And follow up later if we want to replace with fiscalHost.

Reply:
I've changed this to `Amount received for` as suggested. 👍🏽